openflow_input/bsn_tlv

```diff
@@ -1417,3 +1417,26 @@ struct of_bsn_tlv_src_mac_cml : of_bsn_tlv {
     uint16_t length;
     enum ofp_bsn_cml value;
 };
+
+struct of_bsn_tlv_active : of_bsn_tlv {
+    uint16_t type == 192;
+    uint16_t length;
+    of_octets_t value;
+};
```

Comment:
What's the value of tlv_active?

Reply:
The value would be the name of the currently active slave. 
If not clear, we can make this TLV more specific, but will make the TLV specific to this use case.
We could also just use the name TLV and not introduce a new TLV -- so the list of TLVs would be name (of currently active slave), and then one or more groups of name/link_status/fail_count TLVs.
Please let me know what you prefer.

Reply:
Could we have interface_name as generic TLV to carry interface names? tlv_active looks like bool to indicate whether the given entry is active or not.

Reply:
Changed "active" to boolean.  Interface name will be carried by name TLV.